scipy/stats/tests/test_distributions.py

```diff
@@ -955,8 +958,6 @@ def test_pdf_zero(self):
 
 
 class TestGenHyperbolic:
-    def setup_method(self):
-        np.random.seed(1234)
```

Comment:
I didn't expand the entire class - I'm assuming that no global random state is used within it?